env/index.js

```diff
@@ -102,6 +102,18 @@ class Env {
 
     return env === 'testing'
   }
+
+  /**
+   * Determines whether the `NODE_ENV` environment
+   * variable is set to the given `environment`.
+   *
+   * @returns {Boolean}
+   */
+
+  is (environment) {
+    const currentEnvironment = this.get('NODE_ENV', environment).toLowerCase()
```

Comment:
I would prefer to not pass the `environment` variable along as the second parameter to `this.get()` because it will then be the default. In case the `NODE_ENV` isn't set at all, the check `Env.is('local')` returns true even though `NODE_ENV` isn't set to `'local'`.

Reply:
Yep, I didn't figure it out that lodash `get()` method use a default value.